src/org/sosy_lab/java_smt/solvers/opensmt/OpenSmtFormulaManager.java

```diff
@@ -83,4 +83,24 @@ public void appendTo(Appendable out) throws IOException {
       }
     };
   }
+
```

Comment:
This code change is not related to Bitwuzla and should be done in another PR.

Additionally, it might not work with escaped names and string constants in SMTLIB.

Reply:
I just fixed bugs along the way for other solvers (CVC5 is now also affected). Do you want me to remove those and do them separately?

Reply:
I've also opened a issue for the issue you pointed out!

Reply:
I've now removed the changes to other solvers and extracted them into 2 branches, 1 for parsing and 1 for quantifiers